gordo_components/builder/build_model.py

```diff
@@ -368,11 +400,11 @@ def _score_per_tag(y_true, y_pred):
         metrics_dict = {}
         for metric in metrics_list:
             for index, col in enumerate(y.columns):
-                metric_str = metric.__name__.replace("_", "-")
+                metric_str = metric.__name__.replace("_", "_")
                 metrics_dict.update(
                     {
                         metric_str
-                        + f'-{col.replace(" ", "-")}': metrics.make_scorer(
+                        + f'_{col.replace(" ", "_")}': metrics.make_scorer(
```

Comment:
What? Why do this? My comment read that _your_ new method should adhere to how _this_ method use hyphens and not underscores.

Reply:
I think I was confused by what you meant by *this*. So back to how I had it the first time ;)